src/styles/index.scss

```diff
@@ -1,3 +1,118 @@
+* {
+  margin: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: #060b35;
+}
+
 body {
   margin: 0;
+  box-sizing: border-box;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+  box-sizing: border-box;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+}
+
+.card__h1 {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+
+  margin-top: 40px;
+}
+
+.card__p {
+  color: #616070;
+
+  margin-top: 4px;
+}
+
+.card__review {
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-between;
+
+  margin-top: 16px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(/images/star.svg);
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:nth-child(-n+4) {
+  background-image: url(/images/star-active.svg);
+}
+
+.card__price {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  margin-top: 24px;
+}
+
+.card__p1 {
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.card__p2 {
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+}
+
+.card__button {
+  text-decoration: none;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+  width: 166px;
+  height: 40px;
+  border-radius: 5px;
+  color: #fff;
+  background-color: #00acdc;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+
+  margin-top: 16px;
+}
+
+.card__button:hover {
+  box-sizing: border-box;
```

Comment:
The 'box-sizing' property is unnecessary here, as it has already been defined in the .card__button class. Remove this line to make your CSS more efficient.